service/src/main/java/uk/gov/hmcts/reform/fpl/service/docmosis/DocumentConversionService.java

```diff
@@ -0,0 +1,82 @@
+package uk.gov.hmcts.reform.fpl.service.docmosis;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.io.FilenameUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ContentDisposition;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.stereotype.Service;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.RestTemplate;
+import uk.gov.hmcts.reform.fpl.config.DocmosisConfiguration;
+import uk.gov.hmcts.reform.fpl.model.common.DocumentReference;
+import uk.gov.hmcts.reform.fpl.service.DocumentDownloadService;
+import uk.gov.hmcts.reform.fpl.service.UploadDocumentService;
+
+import static com.google.common.net.HttpHeaders.CONTENT_DISPOSITION;
+import static org.springframework.http.MediaType.MULTIPART_FORM_DATA;
+import static uk.gov.hmcts.reform.fpl.model.common.DocumentReference.buildFromDocument;
+
+@Service
+@Slf4j
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class DocumentConversionService {
+    private final RestTemplate restTemplate;
+    private final DocmosisConfiguration configuration;
+    private final DocumentDownloadService documentDownloadService;
+    private final UploadDocumentService uploadDocumentService;
+
+    public DocumentReference convertDocument(DocumentReference document) {
+        //If document is already a pdf, do not convert. Will still 'convert' pdfs that have do not have an extension.
+        if (document.getFilename().contains(".pdf")) {
```

Comment:
use FilenameUtils.getExtension()

maybe method name should be convertToPdf, otherwise why pdf is so special